lib/exec.js

```diff
@@ -150,27 +150,28 @@ function executeDirect(command, args, options, filenameTransform) {
     });
 }
 
-function withNsjailTimeout(args, options) {
-    if (options && options.timeoutMs) {
+export function getNsJailOptions(configName, command, args, options) {
+    options = {...(options || {})};
```

Comment:
This can just be `options = {...options}`. Object-spread on undefined is perfectly legal and simply returns an empty object.

Reply:
oh cool! thanks!